Don't overwrite the user variable CFLAGS< use AM_CFLAGS or nothing.

Index: kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libdvbapi/Makefile.am
===================================================================
--- kaffeine-0.8.7.orig/kaffeine/src/input/dvb/lib/libdvbapi/Makefile.am
+++ kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libdvbapi/Makefile.am
@@ -8,5 +8,3 @@
            dvbfe.c \
            dvbnet.c \
            dvbvideo.c
-
-CFLAGS  = -g -O2 -Wall -Wshadow -Wpointer-arith -Wstrict-prototypes -fPIC
Index: kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libdvben50221/Makefile.am
===================================================================
--- kaffeine-0.8.7.orig/kaffeine/src/input/dvb/lib/libdvben50221/Makefile.am
+++ kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libdvben50221/Makefile.am
@@ -18,4 +18,4 @@ libdvben50221_la_SOURCES = asn_1.c \
            en50221_session.c \
            en50221_transport.c 
 
-CFLAGS  = -g -O2 -DLOG_LEVEL=1 -Wall -Wshadow -Wpointer-arith -Wstrict-prototypes -fPIC
+AM_CFLAGS  = -DLOG_LEVEL=1
Index: kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libucsi/dvb/Makefile.am
===================================================================
--- kaffeine-0.8.7.orig/kaffeine/src/input/dvb/lib/libucsi/dvb/Makefile.am
+++ kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libucsi/dvb/Makefile.am
@@ -15,5 +15,3 @@ libdvbdvb_la_SOURCES = bat_section.c \
            tot_section.c \
            tva_container_section.c \
            types.c
-
-CFLAGS  = -g -O2 -Wall -Wshadow -Wpointer-arith -Wstrict-prototypes -fPIC
Index: kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libucsi/Makefile.am
===================================================================
--- kaffeine-0.8.7.orig/kaffeine/src/input/dvb/lib/libucsi/Makefile.am
+++ kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libucsi/Makefile.am
@@ -16,4 +16,3 @@
        ./dvb/libdvbdvb.la \
        ./atsc/libdvbatsc.la

-CFLAGS  = -g -O2 -Wall -Wshadow -Wpointer-arith -Wstrict-prototypes -fPIC
Index: kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libucsi/mpeg/Makefile.am
===================================================================
--- kaffeine-0.8.7.orig/kaffeine/src/input/dvb/lib/libucsi/mpeg/Makefile.am
+++ kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libucsi/mpeg/Makefile.am
@@ -8,5 +8,3 @@ libdvbmpeg_la_SOURCES = cat_section.c \
            pat_section.c \
            pmt_section.c \
            tsdt_section.c
-
-CFLAGS  = -g -O2 -Wall -Wshadow -Wpointer-arith -Wstrict-prototypes -fPIC
Index: kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libucsi/atsc/Makefile.am
===================================================================
--- kaffeine-0.8.7.orig/kaffeine/src/input/dvb/lib/libucsi/atsc/Makefile.am
+++ kaffeine-0.8.7/kaffeine/src/input/dvb/lib/libucsi/atsc/Makefile.am
@@ -14,4 +14,3 @@
                tvct_section.c \
                types.c

-CFLAGS  = -g -O2 -Wall -Wshadow -Wpointer-arith -Wstrict-prototypes -fPIC
